nucliadb_utils/nucliadb_utils/settings.py

```diff
@@ -189,7 +189,13 @@ class AuditSettings(BaseSettings):
 class NATSConsumerSettings(BaseSettings):
     # Read about message ordering:
     #   https://docs.nats.io/nats-concepts/subject_mapping#when-is-deterministic-partitioning-needed
-    nats_max_ack_pending: int = 1  # required for strict message ordering
+
+    # NATS MaxAckPending controls how many messages can we handle simultaneously
+    # (look at the implementation for how concurrent indexing is implemented)
+    #
+    # NOTE this parameter don't actually change already existing consumers! If
+    # you want to update the value, you should use nats-cli to do so
+    nats_max_ack_pending: int = 10
```

Comment:
Note: 

If you want to play safer, you could leave the default to `1` as it was, and then tweak it to `10` via chart values in stage until we are sure it works fine. And then finally bump this to 10.